docs/SpaceTemplates.rst

```diff
@@ -98,23 +98,23 @@ In which:
-+=========================+==============================+============================================================================+
-| **name**                | ``string``                   | The name of the space template.                                            |
-+-------------------------+------------------------------+----------------------------------------------------------------------------+
-| **visibility**          | ``string``                   | The visibility setting of the space.                                       |
-+-------------------------+------------------------------+----------------------------------------------------------------------------+
-| **registration**        | ``string``                   | The registration setting for the space.                                    |
-+-------------------------+------------------------------+----------------------------------------------------------------------------+
-| **bannerPath**          | ``string``                   | The path of the space banner file inside your extension war.               |
-+-------------------------+------------------------------+----------------------------------------------------------------------------+
-| **permissions**         | ``string``                   | The groups that have permissions to view and use this space template.      |
-+-------------------------+------------------------------+----------------------------------------------------------------------------+
-| **homePageApplication** | ``SpaceApplication``         | The application to use in the home page of a space.                        |
-+-------------------------+------------------------------+----------------------------------------------------------------------------+
-| **applications**        | list of ``SpaceApplication`` | The list of the applications that are installed by default to a new space. |
-+-------------------------+------------------------------+----------------------------------------------------------------------------+
++-------------------------+------------------------------+----------------------------------------------------------------------------------------------+
+| Field name              | Possible value               | Description                                                                                  |
++=========================+==============================+==============================================================================================+
+| **name**                | ``string``                   | The name of the space template.                                                              |
++-------------------------+------------------------------+----------------------------------------------------------------------------------------------+
+| **visibility**          | ``string``                   | The visibility setting of the space. Possible values: "private" or "hidden"                  |
```

Comment:
are you sure ? it's not very clear. Shouldn't it be something like hidden/public ?

Reply:
In fact, for now, we manage 'hidden' or 'private' values only. ("Public" is deprecated, even not included in tests nor in UI)

Reply:
Yes this values are used since a while